pkg/didcomm/packer/packer_opts.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package packer
+
+type packerOpts struct {
+	typHeader string
+	ctyHeader string
+}
+
+// Opts are the packer options.
+type Opts func(opts *packerOpts)
+
+// NewOpt creates a new empty packer option.
+// Not to be used directly. It's intended for implementations of Packer interface
+// Use WithSender() option function below instead.
+func NewOpt() *packerOpts { // nolint // unexported type doesn't need to be used outside of packer package
+	return &packerOpts{}
+}
+
+// TypHeader gets the 'typ' header to be used when packing a message.
+func (pk *packerOpts) TypHeader() string {
+	return pk.typHeader
+}
+
+// CTYHeader gets the 'cty' header to be used when packing a message.
+func (pk *packerOpts) CTYHeader() string {
+	return pk.ctyHeader
+}
+
+// WithTypHeader sets the `typ` header when packing.
+func WithTypHeader(typ string) Opts {
```

Comment:
This PR only uses these two options in unit tests. I assume there will be a followup to expose this as an option to `aries.New()` somehow.

Reply:
currently I decided to have these options exposed on Pack() calls directly, ie  not passing these headers to the packer's New() calls

this means the user has the option to set the `typ` and `cty` protected header when packing which saves the user from creating different packers with different values, one can simply call `Pack(... , packer.WitTypHeader(customTyp), packer.WithCTYHeader(customCTY))`.

there's no need to configure this info at the framework level

Reply:
options are removed